conda/models/version.py

```diff
@@ -12,12 +12,8 @@
 log = getLogger(__name__)
 
 
-def normalized_version(version):
-    """
-    normalized_version() is needed by conda-env
-    It is currently being pulled from resolve instead, but
-    eventually it ought to come from here
-    """
```

Comment:
`conda_env` has long since updated to import from here instead of `conda.resolve`.